scripts/generate_instructions.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 python docs/instructions.py -i -o docs/pages/incidental_occurrence_data-v2.0.0.csv.md incidental_occurrence_data-v2.0.0.csv
+python docs/instructions.py -o docs/pages/incidental_occurrence_data-v3.0.0.csv.md incidental_occurrence_data-v3.0.0.csv
```

Comment:
Running this gives an error
```
ValueError: Template 'incidental_occurrence_data-v3.0.0.csv' not defined; got NoneType
```
Which results in the final docs page having a blank page for Incidental occurrence data v3.0.0.csv 
![image](https://github.com/user-attachments/assets/3fc1374d-bdfc-4288-9de4-0c5b9c676dd2)

Is that ok?

Reply:
nope not good at all. My bad. Will fix.

Reply:
removed